fix (jkube-kit/generator) : Do not add build timestamp to `org.label-schema.build-date` LABEL to better utilize docker cache (#2393) (#2398)
eclipse/jkube: CHANGELOG.md

* Fix #2369: Helm chart apiVersion can be configured
* Fix #2386: Helm icon inferred from annotations in independent resource files (not aggregated kubernetes/openshift.yaml)
* Fix #2397: Default JKube work directory (`jkube.workDir`) changed from `${project.build.directory}/jkube` to `${project.build.directory}/jkube-temp`
* Fix #2393: Remove timestamp from `org.label-schema.build-date` LABEL to utilize docker cache
* Fix #2399: Helm no longer generates default function; broadens support for different value types

_**Note**_:

> - Container Images generated using jkube opinionated defaults would no longer contain full timestamp in `org.label-schema.build-date` label. It would only contain build date.

Reviewer: ```suggestion
- Container Images generated using jkube opinionated defaults no longer contain full timestamp in `org.label-schema.build-date` label. The label contains the build date in the format `yyyy-MM-dd`.
```
